packages/addons/service/librespot/package.mk

```diff
@@ -18,9 +18,9 @@
 ################################################################################
 
 PKG_NAME="librespot"
-PKG_VERSION="685fb4e"
-PKG_SHA256="10ff0e5520e576e72b259f69d37e25ec3e520a4be13cbe9707c68614f24526fc"
-PKG_REV="110"
+PKG_VERSION="431be9e847a584d374ec44e7c138525dd8fbb9d6"
+PKG_SHA256="2e336c5415b6ee6f669e673282daccdd770b15d35dd6d71b39b17dc2aa3424c0"
+PKG_REV="111"
 PKG_ARCH="any"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/librespot-org/$PKG_NAME/"
```

Comment:
as you already touch these files, could you change `https://github.com/librespot-org/$PKG_NAME/` -> `https://github.com/librespot-org/librespot/` (both lines) ? tx